src/System.DirectoryServices.AccountManagement/src/AssemblyInfo.cs

```diff
@@ -0,0 +1,10 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System.Security;
+
+// We need this attributes to match netfx System.DirectoryServices.AccountManagement assembly attributes, if we don't do that we would get: 
+// TypeLoadException: Inheritance security rules violated while overriding member: {member} Security accessibility of the overriding method must match the security accessibility of the method being overriden.
```

Comment:
what is the method being overridden? It's in netcoreapp right? So why does it have a security accessibility on it -- seems like we should remove that?

Reply:
@weshaggard because our tests run against the netstandard implementation we need to add this because the assembly is marked as SecurityCritical ( I don't know where that is coming from). If we don't include this then the tests running against netfx would fail with a TypeLoadException as describe in the file's comment. Any ideas on how to fix it?

Reply:
They way to fix that is to not run this library on .NET Framework :)

Reply:
> They way to fix that is to not run this library on .NET Framework :)

:) I guess we could do that and then whenever we fix: https://github.com/dotnet/corefx/issues/24903 I can add the netfx configuration to the tests. I've already ran them locally against real framework assemblies and they pass :)